components/support/sync-telemetry/metrics.yaml

```
@@ -491,6 +492,424 @@ bookmarks_sync:
       - sync-core@mozilla.com
     expires: never
     lifetime: ping
+
+creditcards_sync:
+  uid:
+    type: string
+    description: >
+      The user's hashed Firefox Account ID.
+    send_in_pings:
+      - creditcards-sync
+    bugs:
+      - https://github.com/mozilla-mobile/android-components/issues/10371
+    data_reviews:
+      - TODO
```

Needs to have the link to Travis' data review+ over here and the other places below too:
```suggestion
      - https://github.com/mozilla-mobile/android-components/pull/10372#issuecomment-850378481
```